.github/labeler.yml

```
@@ -0,0 +1,16 @@
+feature:
+  - head-branch: ['^feature/']
+dependencies:
+  - head-branch: ['^dependencies/']
```

```suggestion
  - head-branch: ['^dependencies/']
```
Perhaps this and `dependency` if possible.